.github/workflows/ci-fork.yml

```diff
@@ -2,7 +2,7 @@ name: "CI Fork and Util tests"
 
 on:
     schedule:
-      - cron: "0 3 * * 1,3,5" # at 3:00 AM UTC on Monday, Wednesday and Friday
+      - cron: "0 1 * * 1,3,5" # at 1:00 AM UTC on Monday, Wednesday and Friday
```

Comment:
Should we run this only once per week?

Since V2 Periphery is getting deprecated.

Reply:
Yeah sure but shouldn't we do that after the next release of Lockup? Until then, its the one with the most usage in production.

Reply:
Good point